lib/install.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var fs = require('fs');
+
+var _ = require('lodash');
+var assert = require('assert-plus');
+var vasync = require('vasync');
+
+/**
+ * Install the enroute routes into the restify server.
+ *
+ * @param {object} opts Options object.
+ * @param {object} opts.enroute The parsed enroute configuration.
+ * @param {object} opts.server The restify server.
+ * @param {function} cb The callback.
+ * @returns {undefined}
+ */
+function install(opts, cb) {
```

Comment:
I've started this convention where the function level cb is always `callback`, and use `cb`, `cb2` for nested async ops. It's nice because you see right away where you exit the top level fn.